core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendUtils.scala

```diff
@@ -142,6 +142,7 @@ private[backends] trait SharedBackendUtils extends Logging with Serializable {
     */
   def getH2OCommonArgs(conf: H2OConf): Seq[String] = {
     new ArgumentBuilder()
+      .add("-allow_clients", "true")
```

Comment:
Just a question: This argument is specified in `getH2OCommonArgs()`. Do we also need it for a client?

Reply:
Good point -> It won't hurt passing this parameter to clients, but not required. I moved the option just to worker nodes